icc/icc_defs.mk

```
@@ -289,7 +332,7 @@ WIN32_VS2013_TEST_CMD       = $(WIN32_TEST_CMD)
 WIN32_VS2013_OPENSSL_TEST_CMD = $(WIN32_OPENSSL_TEST_CMD)
 # This is actually used to build an rc file on Windows
 WIN32_VS2013_ASMOBJS        = icc.res
-WIN32_VS2013_EXTRAS         = 
+WIN32_VS2013_EXTRAS         = PKCS11 PKCS11_PERF
```

```suggestion
WIN32_VS2013_EXTRAS         = $(WIN32_EXTRAS)
```
Better to use previous WIN32 definition here. Check that 'develop' does this so we get less diffs next time.